### DISCUSSION TITLE
Please add [no google Services] depent notification

### DISCUSSION THREAD

**Discussion by Milor123** [Docile-Alligator/Infinity-For-Reddit#906] [Ideas]
Hi dear guys! I am trying using infinity for get my daily reddit notification but i cant, i dont use google services, I use microG

Ive tested al beta and stable versions of infinity, but it dont push my notifications

I would like that you add a option for non google services user, in the notification options

Thank u very mucj for your amazing project, i love this

Tested in A12L CrDroid, poco x3 pro

**Comment by Generator** [comment]
Don't think that any unofficial reddit client uses push notifications check interval. 
You can setup the notification interval Settings -> Notification

**Comment by Milor123** [comment]
It not work, and notification apps only work for messages

**Comment by iTrooz** [comment]
In case you're okay with that solution, you can also enable push notifications in MicroG (you will connect to Google servers tho)

**Comment by Milor123** [comment]
> In case you're okay with that solution, you can also enable push notifications in MicroG (you will connect to Google servers tho)

Nope its not a solution, i use microG for all and i get all google notifications and... check my other post
https://github.com/Docile-Alligator/Infinity-For-Reddit/issues/807 (it problem also occurs with google play devices)

**Comment by andmagdo** [comment]
I use grapheneOS (with no google services or microg installed). I have no issues with my notifications (other than the issue that I am checking if it exists right now).

The app does not seem to need them at all.
